### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by bhargav**
- Move brat rendering inside the Renderers. Right now, I am just passing the `Util` function along in options. Need to find alternate ways to do this.
- Update package configuration
- Have a stable color scheme for less than 20 labels.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  "name": "apelles",
   "version": "0.0.1",
-  "description": "Text Annotation Visualization",
+  "description": "NLP Annotation Visualization Library",
   "main": "index.js",
   "scripts": {
     "lint": "eslint index.js server.js"
   },
-  "author": "Bhargav Mangipudi",
+  "author": "Cognitive Computing Group",
```
Comment (danyaljj) [inline package.json]: Computation? 😆

File: src/renderers/treeViewRenderer.js [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+var utils = require('../utils');
 
 const supportedTypes = ["edu.illinois.cs.cogcomp.core.datastructures.textannotation.TreeView"];
 
-var render = function(viewName, viewType, jsonData, tokenMap) {
+var render = function(viewName, viewType, jsonData, domElement, options) {
+    console.assert(options.hasOwnProperty('brat_util'))
+    console.assert(options.hasOwnProperty('brat_webFontURLs'))
+    console.assert(options.hasOwnProperty('rawText'))
+    console.assert(options.hasOwnProperty('tokenMap'))
```
Comment (danyaljj) [inline src/renderers/treeViewRenderer.js]: nice I didn't know this syntax.

File: package.json (hunk 1 above)
Comment (bhargav) [inline package.json]: Haha. Fixed this. Thanks.

**Comment by danyaljj** [comment]
This is real nice. One slight question/comment: 
Is it possible to remove any trace of "brat" from `index.html` file? (including the heading initializations)
Basically the only remaining thing will be Apelles calls and potentially its initialization(s).

**Comment by danyaljj** [comment]
I guess we're going to address the rest in the future PRs. Merging.

**Pull request merged**